api-tests/features/dl-auth-source-checks.feature

```diff
@@ -31,6 +31,10 @@ Feature: Authoritative source checks with driving licence CRI
       | Attribute | Values          |
       | context   | "check_details" |
     Then I get a 'pyi-driving-licence-no-match-another-way' page response
+    When I submit an 'end' event
+    Then I get an OAuth response
+    When I use the OAuth response to get my identity
+    Then I get a 'P0' identity without a drivingLicence VC
```

Comment:
Do you mean `drivingLicence` or `dcmaw` here? I think dcmaw is the problematic one, since it'll be valid!

Reply:
oh flip, good spot!